internal/services/storage/storage_table_resource.go

```diff
@@ -98,6 +99,12 @@ func resourceStorageTable() *pluginsdk.Resource {
 					},
 				},
 			},
+
+			"resource_manager_id": {
+				Type:        pluginsdk.TypeString,
+				Computed:    true,
+				Description: "The ID of the Table in the Resource Manager format. Useful for cases like IAM that do not use the Data Plane ID.",
```

Comment:
Same as the note in the docs

```suggestion
				Description: "The Resource Manager ID of this Storage Table.",
```